src/app/page/Signup/SignupScene.jsx

```diff
@@ -75,7 +75,7 @@ function SignupScene(props) {
   }
 
   async function updateUser(payload, callback) {
-    payload = getPayloadWithAddress(payload);
+    payload = await getPayloadWithAddress(payload);
     try {
       User.update(payload.uid, payload).then(() => {
```

Comment:
you'll want all of your awaits inside your try catch, otherwise if getPayloadWithAddress fails, it won't get caught by this error handling logic but by the next (if any) error mgmt logic up the tree

Reply:
Thanks, will do!